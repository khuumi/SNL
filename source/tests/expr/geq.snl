938.10 >= 456