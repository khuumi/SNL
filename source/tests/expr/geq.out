Binop (Float 938.1) Geq (Int 456)